server/controllers/converters/locationDetailsConverter.ts

```diff
@@ -7,12 +7,11 @@ import { formatDate, formatDateTime, formatDateTimeISO } from '../../utils/dateH
 import { formatName } from '../../utils/utils'
 import config from '../../config'
 
-export function locationDetailsConverter(
-  prisonerNumber: string,
-  defaultLocationHistoryToDate: Date = new Date(),
-): (location: LocationDetails) => LocationDetailsForDisplay {
-  return location => {
-    if (!location || !prisonerNumber) return null
+export default class LocationDetailsConverter {
```

Comment:
Not sure I'm sold on this needing to be a class.... but I am very call averse in JS